README.md

```diff
@@ -1,26 +1,36 @@
-Bitcoin Core integration/staging tree
+Namecoin Core integration/staging tree
 =====================================
 
-[![Build Status](https://travis-ci.org/bitcoin/bitcoin.svg?branch=master)](https://travis-ci.org/bitcoin/bitcoin)
+https://namecoin.org/
 
-https://www.bitcoin.org
+What is Namecoin? 
+----------------
+
+Namecoin is a decentralized open source information registration and transfer system based on the Bitcoin cryptocurrency.
+
+What does it do?
+----------------
+
+Securely record and transfer arbitrary names (keys).
+Attach a value (data) to the names
+(up to 520 bytes, more in the future).
+Transact namecoins, the digital currency (ℕ, NMC).
+Namecoin was the first fork of Bitcoin and still is one of the most innovative altcoins. It was first to implement merged mining and a decentralized DNS. Namecoin squares Zooko's Triangle!
 
-What is Bitcoin?
+What can it be used for?
 ----------------
 
-Bitcoin is an experimental new digital currency that enables instant payments to
-anyone, anywhere in the world. Bitcoin uses peer-to-peer technology to operate
-with no central authority: managing transactions and issuing money are carried
-out collectively by the network. Bitcoin Core is the name of open source
-software which enables the use of this currency.
+Protect free-speech rights online by making the web more resistant to censorship.
+Access websites using the .bit domain (with TLS/SSL).
+Store identity information such as email, GPG key, BTC address, TLS fingerprints, Bitmessage address, etc.
+Human readable Tor .onion names/domains.
+File signatures, Voting, bonds/stocks,/shares, web of trust, escrow and notary services (to be implemented).
 
-For more information, as well as an immediately useable, binary version of
-the Bitcoin Core software, see https://www.bitcoin.org/en/download.
```

Comment:
I'd prefer to leave this here, with an added note that downloads are not yet posted but will be posted soon.